tests/config/kafka_override.yaml

```diff
@@ -25,6 +25,7 @@ affinity:
             values:
             - linux
 zookeeper:
+  replicaCount: 1
```

Comment:
@yaron2  @youngbupark @wcs1only are we ok with reducing the replicaCount to 1?

Reply:
I'm fine with keeping the existing count too, it's not required for this PR - just improving the workflow time a little bit.

Reply:
this must be ok.